### PR TITLE
Add check target for all the CI

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: centos_ci.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # bundle location
-BUNDLE_VERSION=4.6.1
+BUNDLE_VERSION=4.6.3
```
Comment (cfergeau) [inline centos_ci.sh:4]: Might be a good addition to the `check` target, error out if `grep BUNDLE_VERSION centos_ci.sh` is different from the version we have in the Makefile.

**Review by guillaumerose — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -32,22 +32,7 @@ jobs:
...
-    - run:
-        name: Make cross
-        command: make cross
-    - run:
-        name: Run unit tests
-        command: make test
-    - run:
-        name: Compile integration tests
-        command: make build_integration
+        command: make check
```
Comment (cfergeau) [inline .circleci/config.yml:35]: I did not change circleci configuration on purpose as it was slightly more informative to have the target split. No strong feeling either way though.

**Review by cfergeau — APPROVED** [review]
Dunno how I missed appveyor.yml and centos_ci.sh :) Thanks!

**Comment by anjannath** [comment]
`make cross-lint` is failing in Appveyor
```
/c/gopath/src/github.com/code-ready/crc
GOOS=darwin c:\gopath/bin/golangci-lint run
/usr/bin/sh: c:gopath/bin/golangci-lint: No such file or directory
make: *** [Makefile:165: cross-lint] Error 127
Command exited with code 2
```

**Pull request merged**